core/src/mindustry/content/Items.java

```diff
@@ -43,7 +43,7 @@ public void load(){
 
         titanium = new Item("titanium", Color.valueOf("8da1e3")){{
```

Comment:
Skip to content
Search
Pull requests
Issues
Marketplace
Explore
 
@valeciawolf34 
Anuken
/
Mindustry
398
7.7k1.3k
Code
Issues
5
Pull requests
80
Actions
Security
Insights
Fix
custom-launch-pad-pod-textures @Voz-Duh  75af0e8
Pull Request Tests
on: pull_request
 2
 buildJava14
buildJava14
failed 12 hours ago in 1m 49s
Search logs
3s
4s
5s
1m 36s
Run ./gradlew test desktop:dist
Downloading https://services.gradle.org/distributions/gradle-6.6-bin.zip
.................................................................................................

Welcome to Gradle 6.6!

Here are the highlights of this release:
 - Experimental build configuration caching
 - Built-in conventions for handling credentials
 - Java compilation supports --release flag

For more details see https://docs.gradle.org/6.6/release-notes.html

Starting a Gradle Daemon (subsequent builds will be faster)

> Configure project :
Compiling with build: 'custom build'

> Configure project :android
Scheduling sprite pack.

> Configure project :desktop
Scheduling sprite packing.

> Task :annotations:clearCache
> Task :annotations:compileJava
> Task :annotations:processResources
> Task :annotations:classes
> Task :annotations:jar
> Task :core:clearCache
> Task :core:preGen UP-TO-DATE
/home/runner/work/Mindustry/Mindustry/core/src/mindustry/world/blocks/campaign/LaunchPad.java:204: error: cannot find symbol

> Task :core:compileJava
            TextureRegion region = block.podRegion;
                                        ^
  symbol:   variable podRegion
  location: variable block of type Block
/home/runner/work/Mindustry/Mindustry/core/build/generated/sources/annotationProcessor/java/main/mindustry/gen/LaunchPayload.java:376: error: cannot find symbol
        TextureRegion region = block.podRegion;
                                    ^
  symbol:   variable podRegion
  location: variable block of type Block
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
2 errors


FAILURE: Build failed with an exception.

> Task :core:compileJava FAILED
* What went wrong:

Execution failed for task ':core:compileJava'.
Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
> Compilation failed; see the compiler error output for details.
Use '--warning-mode all' to show the individual deprecation warnings.

See https://docs.gradle.org/6.6/userguide/command_line_interface.html#sec:command_line_warnings
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 35s
6 actionable tasks: 6 executed
Error: Process completed with exit code 1.
0s
0s
Post job cleanup.
0s
Post job cleanup.
/usr/bin/git version
git version 2.29.2
/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :
/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
http.https://github.com/.extraheader
/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :
1s
Cleaning up orphan processes
Terminate orphan process: pid (2685) (java)
usr/bin/git versionhttps://github.com/Anuken/Mindustry/commit/183add4bcb00fce798244aee9215ba1d79576f91https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.2/install.sh